Production/govuk_ios/Extensions/Helper/Redactor/Redactable+Extensions.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+extension Redactable {
+    func redact(text: String) -> String {
```

Comment:
Can you rename this to "redacted" please?
Reason: `redacted` implies the return of a new string, and `redact` implies that it is going to mutate the text that is passed in.